website/project/views/drafts.py

```diff
@@ -22,29 +22,30 @@
 )
 from website import language
 from website.project import utils as project_utils
-from website.project.model import MetaSchema, DraftRegistration
+from website.project.model import MetaSchema, DraftRegistration, DraftRegistrationApproval
 from website.project.metadata.utils import serialize_meta_schema, serialize_draft_registration
 from website.project.utils import serialize_node
 
 get_schema_or_fail = lambda query: get_or_http_error(MetaSchema, query)
 autoload_draft = functools.partial(autoload, DraftRegistration, 'draft_id', 'draft')
 
-# TODO: uncomment to allow submitting drafts for review
-# @must_have_permission(ADMIN)
-# @must_be_valid_project
-# def submit_draft_for_review(auth, node, draft_pk, *args, **kwargs):
-#     user = auth.user
-#
-#     draft = get_draft_or_fail(draft_pk)
-#     draft.is_pending_review = True
-#     draft.save()
-#
-#     REVIEW_EMAIL = Mail(tpl_prefix='prereg_review', subject='New Prereg Prize registration ready for review')
-#     send_mail(draft.initiator.email, REVIEW_EMAIL, user=user, src=node)
-#
-#     ret = project_utils.serialize_node(node, auth)
-#     ret['success'] = True
-#     return ret
+@autoload_draft
+@must_have_permission(ADMIN)
+@must_be_valid_project
+def submit_draft_for_review(auth, node, draft, *args, **kwargs):
+    # TODO(samchrisinger) check that old approval is None or complete
```

Comment:
Part of the requirements for this feature (as I recall) is that if a user submits for approval the old approval is discarded. A user should be warned before she does this. 

It's been a while, but I'd say given that the above feature is in place, the comment can be removed.